src/BuildScriptGenerator/CheckerMessage.cs

```diff
@@ -4,24 +4,19 @@
 // --------------------------------------------------------------------------------------------
 
 using Microsoft.Extensions.Logging;
-using Microsoft.Oryx.BuildScriptGenerator;
 
 namespace Microsoft.Oryx.BuildScriptGenerator
 {
     public class CheckerMessage : ICheckerMessage
     {
-        private readonly LogLevel _level;
```

Comment:
As we discussed earlier in my PR, let's avoid making unrelated changes. A separate clean-up PR would be better for these.